### PR TITLE
Added vision and readmes

### PR DESCRIPTION
**Pull request by ASoTNetworks**
Added README.md files, set up `BW1` to mount NFS share, and added vision playbook.

### CODE REVIEW

File: deploy-hypha-sites/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# deploy Hypha sites
```
Comment (patcon) [inline deploy-hypha-sites/README.md]:
```suggestion
# Deploy Hypha sites
```

**Review by patcon — APPROVED** [review]
READMEs look great, and docs look comprehensive!

FYI I'm unable to test deploy to validate. Not sure whether instructions were supposed to let me provision something local from zero, but I wasn't able to. (It's ok if that's not the intention now)

**Comment by patcon** [comment]
I tried using test-kitchen with its ansible provisions, running against a local virtualbox provider to mock a provision, but kinda got hung up. This is a setup that I have a lot of experience with for testing chef infra, but test-kitchen is agnostic and ansible should theoretically be possible.

In the past, I've used this setup to run a suite of test provisions of chef cookbooks "tasks" across a matrix of multiple supported platforms for every commit to the roles (or cookbooks, the chef equivalent). I had this running on Travis CI on each commit, and it was provisioning a dozen digitalocean instances in parallel, running serverspecs against them, then destroying them. The hours and costs on digitalocean were low. Happy to advise, but I suspect this is a future effort, and not here :)

Even without test-kitchen running automatically in cloud, it's still nice when testing any role/cookbook from scratch (and multiple platforms, if desired) with just:

```
kitchen test deploy-hypha-sites
```

Examples:
https://github.com/patcon/chef-textsecure/blob/master/.kitchen.yml#L48-L70

**Comment by patcon** [comment]
Anyhow, just offering that I have experience with some tools that I could offer support on, if you wanted that sort of infrastructure testing. It feels pretty magic once it's setup, and you can make changes and know everything is as it should be by just checking a PR status msg

File: nginx-reverse-proxy/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# NGINX reverse proxy
+
+The `deploy-nginx-reverse-proxy.yml` playbook will provision a virtual machine and configure the frontend NGINX reverse proxy for backend web servers.
+
+For `staging` environment depoy by running this command:
```
Comment (benhylau) [inline nginx-reverse-proxy/README.md]:
```suggestion
For `staging` environment deploy by running this command:
```

File: nginx-reverse-proxy/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+```
+ansible-playbook -i inventory_staging --ask-vault-pass deploy-nginx-reverse-proxy.yml --extra-vars "staging_env=true"
+```
+
+For `production` environment depoy by running this command:
```
Comment (benhylau) [inline nginx-reverse-proxy/README.md]:
```suggestion
For `production` environment deploy by running this command:
```

**Review by benhylau — APPROVED** [review]
Very cool!

**Pull request merged**